### DISCUSSION TITLE
Next newsletter edition: How to get into the flow

### DISCUSSION THREAD

**Discussion by ettaboyle** [community/community#103148] [Announcements]
## Getting into the flow is magical. :sparkles:

When you achieve flow state, you're more productive, innovative, and happy, according to research. :muscle:
But how do you achieve said flow state? 🤔 

This upcoming GitHub Insider newsletter will provide three key tips for getting in the flow and staying there—empowering you to do your best work. :fire:

[Don't miss it! Sign up now](https://resources.github.com/newsletter/) 

![flow_state_image](https://github.com/community/community/assets/75342807/b1b3da93-05d3-465e-9631-71ff5e21aede)

**Comment by novium2** [comment]
Retiring this post as it is over 6 months old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
